packages/components/pie-button/test/component/pie-button.spec.ts

```diff
@@ -1,11 +1,56 @@
 import { test, expect } from '@sand4rt/experimental-ct-web';
+import type { Locator } from '@playwright/test';
 import { PieButton, ButtonProps } from '@/index';
 
 const props: Partial<ButtonProps> = {
     size: 'large',
     variant: 'primary',
 };
 
+type SizeResponsiveSize = {
+    sizeName: 'xsmall' | 'small-productive' | 'small-expressive' | 'medium' | 'large';
```

Comment:
I think you could do something like `Omit<ButtonSize, 'xlarge'>`?

Reply:
Ah actually it's `Exclude` not `Omit`

Reply:
I suspect we don't have that.